Aptoide/aptoide-client-v8
# Feature/asv 214 notification center empty state

**What does this PR do?**

This PR aims to add an empty state to the notification history fragment. Biggest change this issue had, and solves, is regarding SVG image assets. Using the larger one present here made it look terrible on api level below 21. To solve this, I added a line to build gradle file. From now on however we should use app:srcCompat instead of android:src to load SVG image assets. I don't think previous usages of android:src on smaller scale SVGs became broken but further investigation is needed (this will be done on another ticket);

**Database changed?**

No

**Where should the reviewer start?**

- [ ] InboxFragment.java

**How should this be manually tested?**

Go to your account and check notification history. If you have no notifications present, you should see this;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-214](<https://aptoide.atlassian.net/browse/ASV-214>)


**Code Review Checklist**
